types/react-faux-dom/index.d.ts

```diff
@@ -34,4 +35,6 @@ export interface ReactFauxDomProps {
     isAnimatingFauxDOM(): boolean;
 }
 
-export function withFauxDOM<P>(WrappedComponent: any): React.ClassicComponentClass<P & ReactFauxDomProps>;
+type Omit<T, K extends keyof T> = Pick<T, ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never, [x: number]: never })[keyof T]>;
```

Comment:
Please don't write Omit like that. Update the typescript to `TypeScript 2.8` and use [Predefined Conditional Types](http://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html), i.e. `Exclude<T, U>`

Reply:
OK, updated.